src/events.ts

```diff
@@ -1,10 +1,14 @@
 import { Functor$create, FunctorWith1Arg, FunctorItem } from "./functor";
 import { hasOwnProperty } from "./helpers";
 
-export class EventObject {
-	stopPropagation(): void {
-		// TODO: Implement 'stopPropagation()'?
-		throw new Error("Method 'stopPropagation' is not implemented.");
+export interface EventObject {
+	preventDefault(): void;
```

Comment:
I wouldn't mind rolling this change back, since I only added it to support the unit tests. However, it seems simple enough. My only concern would be whether it creates some ambiguity or confusion for other events, since they wouldn't necessarily do anything in response to `preventDefault` being called.